sources/gerritDB/index.js

```diff
@@ -162,125 +173,127 @@ var gerritDB = (function () {
 
     function getTopReviewers (filterOption, callback) {
         var filterDate = getFilterDate(filterOption);
-
-        var query = "select full_name,"
-            + " dest_project_name,"
-            + " patch_set_approvals.value"
-            + " from patch_set_approvals"
-            + " inner join changes on patch_set_approvals.change_id = changes.change_id"
-            + " inner join accounts on patch_set_approvals.account_id = accounts.account_id"
-            + " where date_format(patch_set_approvals.granted, '%Y-%m-%d') >=" + "'" + filterDate + "'"
-            + " and patch_set_approvals.category_id = 'Code-Review'"
-            + " and changes.owner_account_id != accounts.account_id"
-            + " and full_name != 'Jenkins'";
-
-        doQuery(query, function (queryResult) {
+        var query =
+            `SELECT full_name,
+                dest_project_name,
+                patch_set_approvals.value
+            FROM patch_set_approvals
+                INNER JOIN changes ON patch_set_approvals.change_id = changes.change_id
+                INNER JOIN accounts ON patch_set_approvals.account_id = accounts.account_id
+            WHERE DATE_FORMAT(patch_set_approvals.granted, '%Y-%m-%d') >= '${filterDate}'
+                AND patch_set_approvals.category_id = 'Code-Review'
+                AND changes.owner_account_id != accounts.account_id`;
```

Comment:
@mustafaersahin - as @lihorecoco said, the end result is being filtered elsewhere. by convention, we tend to get as raw data as possible from SQL's so that we can do whatever we want afterwards. your SQL changes, on the other hand, requires Jenkins (32) to be in the SQL since you're doing calculation in the query.